src/java/htsjdk/samtools/util/ComparableTuple.java

```diff
@@ -0,0 +1,20 @@
+package htsjdk.samtools.util;
+
+/**
+ * Created by farjoun on 1/17/16.
+ */
+public class ComparableTuple<A extends Comparable<A>, B extends Comparable<B>> extends Tuple<A, B> implements Comparable<ComparableTuple<A, B>> {
```

Comment:
Could you add a class comment explaining that these will be sorted by A first and then B?